tt_eager/tt_dnn/op_library/transformer_tms/transformer_tms.hpp

```diff
@@ -216,6 +224,44 @@ inline Tensor ssm_1d_sum_reduce(const Tensor &input_tensor_a, const MemoryConfig
     return output_tensors.at(0);
 }
 
+struct SSMPrefixScan {
+    MemoryConfig output_mem_config;
+    DataType output_dtype;
+    MathFidelity math_fidelity;
+
+    const uint32_t HIDDEN_SIZE = 5120;
+    const uint32_t LATENT_SIZE = 32;
```

Comment:
I think these should not have a default value and we pass them in during op creation.

Reply:
These actually aren't used anywhere, because the OP is general enough to handle any hidden/latent size. I've removed them.